samples/cellular/nrf_cloud_multi_service/src/cloud_connection.c

```diff
@@ -551,6 +551,31 @@ static int setup_cloud(void)
 	return 0;
 }
 
+static void wait_for_credentials(void) {
+#if defined(CONFIG_NRF_CLOUD_CHECK_CREDENTIALS)
```

Comment:
do you plan to disable `CONFIG_NRF_CLOUD_CHECK_CREDENTIALS` in other build configurations?
if not, this check can be removed.

Reply:
I don't have any such plans, but I thought being able to disable it might be useful for testing. But thinking on it, in those situations it is probably just as easy for us to temporarily remove the check manually. So I'll remove the guards.